Lib/abc.py

```diff
@@ -82,13 +83,22 @@ def my_abstract_property(self):
 
 
 try:
-    from _abc import (get_cache_token, _abc_init, _abc_register,
-                      _abc_instancecheck, _abc_subclasscheck, _get_dump,
-                      _reset_registry, _reset_caches)
+    from _abc import (
+        get_cache_token,
+        _abc_init,
+        _abc_register,
+        _abc_instancecheck,
+        _abc_subclasscheck,
+        _get_dump,
+        _reset_registry,
+        _reset_caches,
+    )
```

Comment:
Please could you revert the Black formatting?

Whilst I personally like it, it's a bigger conversation than this PR and we'd like to otherwise avoid churn. For example, it's not easy to see if you've made any changes here.

Reply:
Whoops! Sorry, thanks for the catch.